manager/container.go

```diff
@@ -324,6 +328,15 @@ func (c *containerData) GetProcessList(cadvisorContainer string, inHostNamespace
 			cgroupPath = cgroup
 		}
 
+		var fdCount int
+		dirPath := path.Join(rootfs, "/proc", strconv.Itoa(pid), "fd")
+		fds, err := ioutil.ReadDir(dirPath)
+		if err != nil {
+			glog.V(4).Infof("error while listing directory %v :%s", dirPath, err)
```

Comment:
can the error message include that we are listing the directory to get fd count?
E.g.: `error while listing directory %s to measure fd count: %v`